security/advancedtls/crl.go

```diff
@@ -303,24 +303,30 @@ func fetchCRL(c *x509.Certificate, crlVerifyCrt []*x509.Certificate, cfg Revocat
 		if crl == nil {
 			return nil, fmt.Errorf("no CRL found for certificate's issuer")
 		}
+		if err := verifyCRL(crl, nil, crlVerifyCrt); err != nil {
```

Comment:
is the `nil` `rawIssuer` arg actually used at all in this fn?

Reply:
Not for this functionality, let's discuss one previous usage offline

Reply:
removed